extensions/wikia/Email/styles/specialSendEmail.scss

```diff
@@ -0,0 +1,12 @@
+.WikiaForm {
+
+	margin-bottom: 15px;
+
+	input[type="submit"] {
+		margin-top: 5px;
+	}
+
+	.input-group {
+		clear: none;
+	}
+}
```

Comment:
So I've seen IDE's complain about that. From what I can tell it's not in our guidelines, is it just an overall best practice for any source code?